### ISSUE TITLE
Auto Rotation

### ISSUE THREAD

**Issue by Lucky390163** [nova-video-player/aos-AVP#748]
### Problem description

Auto rotation is not working in nova player it cannot rotate even the option is on.

### Steps to reproduce the issue

Play the video.

### Expected behavior

Not good.

### Your phone/tablet/androidTV model

SM-M127G

### Operating system version

12

### Application version and app store

6.0.81 - 20221108.2142

**Comment by mitch133** [comment]
Works for me with 6.0.81 + Galaxy S21 (android 13)

**Comment by Lucky390163** [comment]
But not in my device i tried all rotation methods.

**Comment by courville** [comment]
@Lucky390163, perhaps an accelerometer issue. Have you tried to reboot the device?

**Comment by Lucky390163** [comment]
Yes 5 time while in other video players app working too smoothly.

**Comment by Lucky390163** [comment]
And Sir Nova video player is consuming more batter than i expected it consumes 3 percent battery in 15 minutes while my Samsung video library player consumes 2 percent battery in 35 minutes.

Device - SM-M127G battery - 6000mah

**Comment by courville** [comment]
Let's not mix issues please and focus on the first one.
In nova video player, can you please check that in the player menu (i.e. while playing the video), you have the option "Automatic screen rotation" enabled?
Of course you need to have global phone setting to autorotate too (but that I trust is ok).

**Comment by Lucky390163** [comment]
> Let's not mix issues please and focus on the first one.
> In nova video player, can you please check that in the player menu (i.e. while playing the video), you have the option "Automatic screen rotation" enabled?
> Of course you need to have global phone setting to autorotate too (but that I trust is ok).

Yes.

**Comment by Lucky390163** [comment]
> > Let's not mix issues please and focus on the first one.
> > In nova video player, can you please check that in the player menu (i.e. while playing the video), you have the option "Automatic screen rotation" enabled?
> > Of course you need to have global phone setting to autorotate too (but that I trust is ok).
> 
> Yes.I tried system auto rotate not works in nova place while vlc, mx pro can rotate easily.

**Comment by Lucky390163** [comment]
Yes.I tried system auto rotate not works in nova place while vlc, mx pro can rotate easily.

**Comment by courville** [comment]
OK I did extensive tests on a samsung s22 and I am sorry to report that on this device I am unable to reproduce your regression behavior. For now, I have no way to reproduce/debug.

**Comment by Lucky390163** [comment]
Can i send u a video recording of this.

**Comment by courville** [comment]
@Lucky390163, yes this could help indeed. Thank you.

**Comment by Lucky390163** [comment]
-

**Comment by Lucky390163** [comment]
Sending u on email github cannot exceed size.

**Comment by courville** [comment]
you can send it to software at courville dot org

**Comment by Lucky390163** [comment]
There are many options i cannot find sending option.

**Comment by mschumacher69** [comment]
Share a link via google photos.

**Comment by Lucky390163** [comment]
I will sen you on reddit.

**Comment by Lucky390163** [comment]
Please add a option to use screen rotation in playing section.

**Comment by courville** [comment]
@Lucky390163 can you please share the link towards the video (not on reddit)?

**Comment by Lucky390163** [comment]
On reddit?

**Comment by Lucky390163** [comment]
![Screenshot_20221212-122539_Reddit.jpg](https://user-images.githubusercontent.com/115852216/206980671-e1325d27-3b6f-49a4-8b73-086d6bc83672.jpg)

**Comment by Lucky390163** [comment]
Sent.

**Comment by courville** [comment]
@Lucky390163 thanks for sharing the video, it is clear now. Cf. https://www.reddit.com/r/NovaVideoPlayer/comments/zjqj2g/auto_rotation_not_working_correctly/
When you start the video with rotation locked if you change the setting while playing the video it will not change the status checked at start of the video. I probably need to change this. It would work if before starting the video you have rotation not locked in both settings.

**Comment by Lucky390163** [comment]
Please can u add a manual screen rotation button.

**Comment by Lucky390163** [comment]
Please make rotation just like vlc without changing the actual orientation . Otherwise iam not able to handle my phone rotation in this manner.It is very difficult.
